translation/locale.rst

```diff
@@ -26,6 +26,13 @@ it::
             $request->setLocale($locale);
         }
 
+.. note::
+
+    Using a listener like this requires that it is called **before** 
+    LocaleListener attempts to access it. In Symfony Framework it is called
+    with priority 8 by default. You will need to set your event listener
```

Comment:
Better not to write the priority number if it changes
Just talk about defining the listener with a higher priority than the locale one

Reply:
Updated accordingly. Thanks.